### ISSUE TITLE
Rewrite the game using a consolidated game engine

### ISSUE THREAD

**Issue by nathanfabiano** [claudiosegala/regalia#4]
Since we will continue the project for a future publication, I believe we would have many benefits in rewriting the game with Unity3D or Unreal.
We will waste a little time rewriting, but we will save a lot of time adding new contents to the game in the future.

**Comment by claudiosegala** [comment]
@ramkury what do you think? I kinda like doing it in SDL because is more challenging but if you guys want to change the game engine, I am ok with it.

**Comment by ramkury** [comment]
I agree with @claudiosegala. Maybe we could create a different project to make a similar game using a commercial game engine, sharing the assets with our current project, but I feel that this project should go on using pure SDL.
I think the goal of this project should not be to make a commercial game, at least not for a long time. We still have a long way to go, and if we happen to achieve a sufficient level of quality, then we could start thinking about publishing the game somewhere.

**Comment by claudiosegala** [comment]
@ramkury and @gustavohfc can I close this issue then?
